src/pages/releases.mdx

```diff
@@ -52,7 +52,7 @@ MariaDB requires the use of the [mariadb][mariadb-npm] npm package.
 
 | Sequelize   | [MariaDB][mariadb] | [mariadb (npm)][mariadb-npm] |
 |-------------|--------------------|------------------------------|
-| 7 (alpha)   | >=10.3             | >= 3.0.0 [^⚠️2]              |
+| 7 (alpha)   | >=10.4             | >= 3.0.0 [^⚠️2]              |
```

Comment:
These should be aligned with what is in our mariadb/oldest folder, which is 10.4.6